### PR TITLE
2.2.46 legible text on windows (bye Lato)

### PR DESCRIPTION
**Pull request by 2is10**

### CODE REVIEW

File: packrat/html/metro/pane_thread.html [hunk 1]
```diff
@@ -2,21 +2,20 @@
...
+    A conversation with{{!
+  }}{{^numRecipients}}:
+      {{#recipients}}
+        <span class="kifi-thread-participant">{{firstName}} {{lastName}}</span>
+      {{/recipients}}
+    {{/numRecipients}}
+    {{#numRecipients}}
+      {{numRecipients}} people:
+      <div class="kifi-thread-participants">
+      {{#recipients}}{{!
```
Comment (andrewconner) [inline packrat/html/metro/pane_thread.html:14]: Hah, interesting formatting hack.

File: packrat/styles/metro/slider2.css [hunk 2]
```diff
@@ -126,7 +99,7 @@
   right: 0;
   height: 68px;
   width: 142px;
-  font: 12px Lato,sans-serif;
+  font: 12px "Helvetica Neue",arial,sans-serif;
```
Comment (andrewconner) [inline packrat/styles/metro/slider2.css:102]: Will we use Helvetica webfonts as well? Linotype offers it at a decently inexpensive rate: http://www.linotype.com/1266/NeueHelvetica-family.html?site=webfonts I'll ping Danny.

Comment (2is10) [inline packrat/styles/metro/slider2.css:102]: Danny's happy with Arial on Windows. Macs have Helvetica Neue. I don't think we need a webfont. Danny doesn't trust any webfonts now on his PCs, and I don't blame him. I think the Lato files did have good hinting. If you can show him a webfont that looks good at 12px on his PCs, then he may be convinced.

Comment (andrewconner) [inline packrat/styles/metro/slider2.css:102]: Could we, perhaps, use a larger size than 12px? :)

Comment (2is10) [inline packrat/styles/metro/slider2.css:102]: Facebook and Pinterest use 11px. We just increased ours from 11px to 12px. We might go to 13px at some point, but certainly no larger than that for potentially long text.

**Pull request merged**